pkg/listener/http/http_listener.go

```diff
@@ -80,6 +80,22 @@ func (ls *HttpListenerService) Start() error {
 	return nil
 }
 
+func (ls *HttpListenerService) Close() error {
+	return ls.srv.Close()
+}
+
+func (ls *HttpListenerService) ShutDown() error {
+	//TODO implement me
+	panic("implement me")
```

Comment:
just remove panic, leave it empty implement? there are many cases

Reply:
ok